DEVELOPMENT.rst

```diff
@@ -0,0 +1,52 @@
+Developing on edx-sga
+=====================
+
+Setup (including devstack setup)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. `Install vagrant <http://docs.vagrantup.com/v2/installation/>`_.
+#. `Install virtualbox <https://www.virtualbox.org/wiki/Downloads>`_.
+#. Set up devstack::
+
+    mkdir devstack
+    cd devstack
+    curl -L https://raw.githubusercontent.com/edx/configuration/master/vagrant/release/devstack/Vagrantfile > Vagrantfile
+    vagrant plugin install vagrant-vbguest
+    vagrant up
+    cd themes/
```

Comment:
Consider moving this step into the next section since it's a prereq for those steps.